test/config.js

```diff
@@ -8,4 +8,19 @@ lab.experiment('Ensure config is correct', () => {
       require('../server/config')
     }).not.to.throw()
   })
+
```

Comment:
I went a bit off piste here, but getting the code coverage down is no bad thing.